apps/prairielearn/src/migrations/20230922214925_user_sessions__create.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE
+  user_sessions (
+    id BIGSERIAL PRIMARY KEY,
+    session_id TEXT NOT NULL,
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    expires_at TIMESTAMP WITH TIME ZONE NOT NULL,
+    user_id BIGINT REFERENCES users (user_id) ON UPDATE CASCADE ON DELETE CASCADE,
+    data JSONB NOT NULL,
+    UNIQUE (session_id)
+  );
+
+CREATE INDEX IF NOT EXISTS user_sessions_user_id_updated_at_idx ON user_sessions (user_id, updated_at);
+
+CREATE INDEX IF NOT EXISTS user_sessions_updated_at_idx ON user_sessions (updated_at);
```

Comment:
Expected access patterns:

- Find all recent sessions for a given user
- Find all recent sessions

I'm less sure if the latter index is required - maybe it's completely unnecessary?

Reply:
Yeah, feels like its not needed to me.

Reply:
You took the `pl_` naming out of the table name. You don't anticipate using this library with other Prairie stuff, potentially in the same database?

Reply:
Sounds good, I'll remove it.

Reply:
Removed in f4a59fe.